content/blog/2020-sponsor-announcement.md

```diff
@@ -0,0 +1,39 @@
+Title: Call for Sponsors is now open for PyCon India 2020
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
+Category: 2020 
+Slug: announce-sponsorsership
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India. 
+
+While the PyCon team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
+
+
+The entire organizing team and the Python community at large are extremely thankful to all the sponsors for supporting us throughout the years. Without your support, we wouldn't have been able to even think about making the conference bigger, better, and more accessible for everyone every year. Many of you must already be eager to come forward and book your slot, let me tell you a little about how sponsoring the conference can be beneficial for your organization:
+
+## Put your business in the spotlight
+
+Increase your brand exposure and elevate your business identity within the Python community. It is your chance to stand out among the conference attendees by prominently displaying your products and marketing material.
+
+## Give Back to the Community
+
+Establish your goodwill with the community and show the community you’re a reliable partner, willing to support the ecosystem and help it thrive in such challenging times. Sponsorship is a way of giving back to the community and thanking them for their support. Your sponsorship helps PyCon India make the conference affordable and accessible to the widest possible audience.
+
+## Low-Cost Marketing
+
+Instead of spending a fortune on traditional advertisements, take a look at our reasonably priced sponsorship slabs and pick one - letting you be discovered and known to the diverse and widespread Python community at maybe half (or even less!) of your marketing budget.
+
+## Hiring and Business Opportunities
+
+Get matched with folks who could become your next potential clients or employees. The conference has a history of attracting smart folks and businesses from various backgrounds. Get connected, make your pitch, and find the best programmers or clients to join your organizations!
+
+## Flexibility
+
+A sponsorship is a two-way partnership; we want both parties to be happy and gain something from this opportunity. If you have something specific you require from us, please don’t hesitate to ask! We are more than happy to work with you and tailor the sponsorship benefits to suit your needs.
+
+## Startup Discount
```

Comment:
There is no Startup Discount slab, it is just a Startup slab. Maybe reword to "Startup slab"?

Reply:
```suggestion
## Startup Slab
```